zeppelin-web/src/index.html

```diff
@@ -100,6 +100,20 @@
     <![endif]-->
     <!-- endbuild -->
 
+    <script type="text/x-mathjax-config">
```

Comment:
Do you think we can move this part in a angular way like directive or something similar?